src/components/designers/mock-data.json

```diff
@@ -0,0 +1,33 @@
+{
+  "designers": [
+    {
+      "id": "designerSoujanya",
+      "last_name": "Bagi",
+      "yoe": "2",
+      "incompleteUserDetails": false,
+      "roles": {
+        "member": true
+      },
+      "first_name": "Soujanya",
+      "company": "Real Dev Squad",
+      "linkedin_id": "soujanyabagi",
+      "username": "soujanya",
+      "isMember": true,
+      "img_url": ""
+    },
+    {
+      "id": "designerArushi",
+      "last_name": "",
+      "yoe": "2",
+      "incompleteUserDetails": false,
+      "roles": {
+        "member": true
+      },
+      "first_name": "Arushi",
+      "company": "Real Deb Squad",
+      "username": "arushi",
+      "isMember": true,
+      "img_url": ""
+    }
+  ]
+}
```

Comment:
Why are we mocking the user data? Don't we have their data in members API?
Designer's profile will be displayed on members page only, They won't have their own profile page, for now right?
Shouldn't we show their profile page also?

Reply:
We will be showing their profile page as well.
Right now we do not have their data set up in the DB Yet. So @ankushdharkar suggested going with the mock data for designers.
We will be later setting things up for them from the ground up.

Reply:
ok, then it look good :+1: